scripts/globals/dealer_moogle.lua

```diff
@@ -3348,15 +3348,15 @@ local getItemSelection = function(player, list, idx, idxAlt1, idxAlt2)
         if debug.ENABLED and not debug.SHOWITEM then
             item = 0
         else
-            --- TODO: Refactor function to always return table, and find better way to determine behavior based on list type.
+            --- TODO: Find better way to determine behavior based on list type.
             ---@diagnostic disable-next-line: cast-local-type
             item = itemList[list][idxAlt1][idxAlt2]
         end
 
         if list == 12 then  -- Item, Quantity
-            --- TODO: Refactor function to always return table, and find better way to determine behavior based on list type.
+            --- TODO: Find better way to determine behavior based on list type.
             ---@diagnostic disable-next-line: cast-local-type
-            item = { item } -- Tabling here to save 100 pairs of { }
+            item = item -- Tabling here to save 100 pairs of { }
```

Comment:
Is that comment here obsolete?

Reply:
Ah, you beat me - yeah I just removed it